### PR TITLE
fix: [Chart widget] Remove unnecessary semicolon in the custom echart config

### PR DESCRIPTION
**Pull request by sbalaji1192**
#### PR fixes following issue(s)
Fixes https://github.com/appsmithorg/appsmith/issues/30444

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Chore (housekeeping or task changes that don't impact user perception)
- This change requires a documentation update
>
>
>
- [ ] Manual
- [ ] JUnit
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

## Summary by CodeRabbit

- **New Features**
	- Introduced a new migration function to enhance the customization of chart widgets.
- **Bug Fixes**
	- Adjusted the suffix for a property in chart widgets to ensure proper configuration.
- **Tests**
	- Added tests for the new migration function, ensuring it performs as expected.

### CODE REVIEW

**Review by rajatagrawal — APPROVED** [review]

File: app/client/src/widgets/ChartWidget/widget/propertyConfig.ts [hunk 1]
```diff
@@ -85,7 +85,7 @@ export const contentConfig = () => {
             wrapperCode: {
               prefix: "{{ ((chartType) => ( ",
               suffix: (widget: WidgetProps) =>
-                `))(${widget.widgetName}.chartType); }}`,
+                `))(${widget.widgetName}.chartType) }}`,
```
Comment (brayn003) [inline app/client/src/widgets/ChartWidget/widget/propertyConfig.ts:88]: This doesn't seem to be related to migration. Is this intentional?

Comment (sbalaji1192) [reply app/client/src/widgets/ChartWidget/widget/propertyConfig.ts:88]: Yes. This is the main change that we want to do. We are doing the migration to make this change in the existing widgets.

**Pull request merged**